app/build.gradle

```diff
@@ -49,6 +49,8 @@ android {
 
 dependencies {
 
+    def compose_version = "1.0.5"
```

Comment:
I kept mine at 1.0.1, we might run into compatibility issues but it's probably a good idea to move to latest version